conman/routes/models.py

```diff
@@ -116,6 +117,18 @@ def reset_originals(self):
         self._original_parent_id = self.parent_id
         self._original_slug = self.slug
 
+    def clean(self):
+        """Ensure only one Root Route can exist."""
+        is_root = self.parent_id is None
+        if is_root:
+            try:
+                root = Route.objects.get(parent=None)
+            except Route.DoesNotExist:
+                pass
+            else:
+                if root != self:
+                    raise ValidationError('Only one Route can exist.')
```

Comment:
This seems ok to me, especially with #68. That said, is it required to call `clean()` from within `save()`?